getOption.m

```diff
@@ -7,18 +7,18 @@
 end
 
```

Comment:
Also adjusted for the motion dataset. 
Some data paths are mine, which will be edited by every user.